### PR TITLE
Update script-topics.rpy

### PR DESCRIPTION
**Pull request by Waffiez**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4560,6 +4560,31 @@ label monika_ageinf:
     m "But I don't believe that my life can be summed up in a number."
     m 5a "So I'll always love you for all eternity, [player]. Just remember that."
     return
+    
+    init 5 python:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove indentation before the `init` block. `init` python blocks **cannot** have any indentation

**Comment by Rune0n** [comment]
Please fix the branch conflicts.

**Comment by ThePotatoGuy** [comment]
Conflicts, fixed @Rune0n

**Comment by Rune0n** [comment]
Requesting rework on Law topic. It seems to contradict itself when it flies from "intentional" to "without knowing", and then discusses the purpose of some rules. I recommend you focus on only one or two of those things at a time.

**Pull request closed without merging**